client/src/reference-provider/parse.ts

```diff
@@ -2,6 +2,7 @@ import linenumber = require('linenumber');
 import fs = require('fs');
 import { getIncludes, getMatchingVariable, normalizeAliasTemplate } from '../template-utils';
 import { VariablePathDescription, VariablePathMap } from '../interfaces';
+import { nextTick } from 'node:process';

```

Comment:
Remove this, it doesn't seem to be used.

Reply:
I dont even know why that was added.